src/queries/page.server.ts

```diff
@@ -6,7 +6,7 @@ export const fetchGetPage = async (
   input: Parameters<typeof pageModel.getPage>[0],
   queryClient: QueryClient,
 ) => {
-  const key = ["getPage", input.page, input]
```

Comment:
I am concerned that different parameters will be cached as the same content

Reply:
You are right, with the key **getPage:input.page**, if two users have the same "pageSlug" the cache will be changed ... to add _input_  is a solution, we can do it better,
the key will be **getPage:input.site:input.page**, it will resolve the problem. in my opinion, the object **input** will not be used in the key.